dev-vcs/svn2cl/svn2cl-0.14.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Create a GNU-style ChangeLog from subversion's svn log --xml output"
+HOMEPAGE="https://arthurdejong.org/svn2cl/"
+SRC_URI="${HOMEPAGE}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+
+RDEPEND="dev-libs/libxslt
+	dev-vcs/subversion"
+
+src_prepare() {
+	# the wrapper script looks for the xsl files in the
+	# same directory as the script.
+	eapply -p0 "${FILESDIR}"/0.9-wrapper.patch
```

Comment:
Could you update the patch to apply with `-p1` while at it? Should be sufficient to add `a/` `b/` prefixes before paths.

Reply:
Done.